hooks/SaveManager.cpp

```diff
@@ -2,13 +2,13 @@
 
 void dSaveManager_set_BodyColor(uint8_t value, MethodInfo* method)
 {
-	std::cout << "SaveManager_set_BodyColor " << value << std::endl;
 	SaveManager_set_BodyColor(value, method);
 }
 
 uint8_t dSaveManager_get_BodyColor(MethodInfo* method)
 {
-	uint8_t value = SaveManager_get_BodyColor(method);
-	std::cout << "SaveManager_get_BodyColor " << value << std::endl;
-	return value;
+	if (State.AntiBan)
```

Comment:
Why not `return (State.AntiBan) ? 20 : SaveManager_get_BodyColor(method);`?

Also is there an enum for the color?